es/resource_elasticsearch_opendistro_role.go

```diff
@@ -43,6 +43,14 @@ func resourceElasticsearchOpenDistroRole() *schema.Resource {
 							},
 							Set: schema.HashString,
 						},
+						"dls": {
```

Comment:
I didn't realize that we abbreviated `fls` before, it's a bit confusing, can we use `document_level_security` here?